Demo/Demo/DemoViewControllerContainer.swift

```diff
@@ -0,0 +1,103 @@
+import FinniversKit
+
+class DemoViewControllerContainer<View: UIView>: UIViewController {
+
+    public private(set) var containmentOptions: ContainmentOptions
+    private var dismissType: DismissType
+    private var preferredInterfaceOrientation: UIInterfaceOrientationMask = .all
+    private let constrainToBottomSafeArea: Bool
+    private let constrainToTopSafeArea: Bool
+    private var bottomSheet: BottomSheet?
+
+    public init(dismissType: DismissType = .doubleTap,
+                containmentOptions: ContainmentOptions = .none,
+                supportedInterfaceOrientations: UIInterfaceOrientationMask = .all,
+                constrainToTopSafeArea: Bool = true,
+                constrainToBottomSafeArea: Bool = true) {
+        self.dismissType = dismissType
+        self.containmentOptions = containmentOptions
+        self.preferredInterfaceOrientation = supportedInterfaceOrientations
+        self.constrainToBottomSafeArea = constrainToBottomSafeArea
+        self.constrainToTopSafeArea = constrainToTopSafeArea
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    var childViewController: DemoViewController<View>?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .black
+        let viewController = DemoViewController<View>(
+            dismissType: dismissType,
+            containmentOptions: containmentOptions,
+            supportedInterfaceOrientations: supportedInterfaceOrientations,
+            constrainToTopSafeArea: constrainToTopSafeArea,
+            constrainToBottomSafeArea: constrainToBottomSafeArea)
+        addChild(viewController)
+        view.addSubview(viewController.view)
+        viewController.didMove(toParent: self)
+        childViewController = viewController
+
+        if let deviceIndex = State.lastSelectedDevice, deviceIndex < Device.all.count {
+            let device = Device.all[deviceIndex]
+            viewController.view.frame = device.frame
+            viewController.view.autoresizingMask = device.autoresizingMask
+            setOverrideTraitCollection(device.traits, forChild: viewController)
+        }
+
+        NSLayoutConstraint.activate([
+            viewController.view.topAnchor.constraint(equalTo: view.topAnchor),
+            viewController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            viewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            viewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            viewController.view.centerYAnchor.constraint(equalTo: view.centerYAnchor),
```

Comment:
I guess there is no need to add center X/Y constraints since there are top, bottom, leading and trailing.